src/NServiceBus.AcceptanceTests/Recoverability/When_transactions_off_and_delayed_retries_enabled.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace NServiceBus.AcceptanceTests.Recoverability
+{
+    using System;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using NUnit.Framework;
+
+    public class When_transactions_off_and_delayed_retries_enabled : NServiceBusAcceptanceTest
```

Comment:
I think we can make this a "core only" test?

Reply:
This is something I asked about in Slack. This one and the two others seem like they could be moved to the core folder. I'll move them.